tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -310,9 +338,14 @@ def agent_test_suite(self, node: Node, variables: Dict[str, Any], log_path: str,
         """
         self._set_context(node, variables, log_path, log)
 
-        test_suite_success = True
-
         with _set_thread_name(self.context.image_name):  # The thread name is added to self._log
+            self._log.info(
```

Comment:
I move this message here from the initialization function. We set the thread name for the log after the initialization, so it was missing from the log for this message